bouncer/src/repo/manipulator/modelconvertor/import/odaHelper/data_processor_rvt.h

```diff
@@ -141,6 +137,8 @@ namespace repo {
 					OdBmDatabasePtr database;
 					OdBmSampleLabelUtilsPE* labelUtils = nullptr;
 				};
+
+				bool TryConvertMetadataEntry(OdTfVariant& metaEntry, OdBmLabelUtilsPEPtr labelUtils, OdBmParamDefPtr paramDef, OdBmDatabase* database, OdBm::BuiltInParameter::Enum param, repo::lib::RepoVariant& v);
```

Comment:
This was a bit of an headache. I wanted to keep it consistent between the importers, but also consistent with the existing style. I would also have preferred to make them inaccessible from the outside, but that makes testing harder of course. So I originally settled on these global functions, since they were as unbound to a class as most other functions belonging to the data processors but, because they were global, still accessible for testing.

The Shims were a good idea because it would allow me to keep them private, but not workable with NWD because the functions that use the converter are all unbound so they can't access private or protected members of it, meaning that it would need to be public again, negating the benefit of the Shim.

In the end, I have now settled on public statics. Consistent between the processors, available for testing, and still bound to a class. Probably the best compromise.

[cc39d9b](https://github.com/3drepo/3drepobouncer/pull/694/commits/cc39d9b39e5ecdac8cff47d0ab1e902b48fc1210)

Reply:
Also, database has been removed:
[2db1b1b](https://github.com/3drepo/3drepobouncer/pull/694/commits/2db1b1bb73354dc28c307cf3340ff0551bb2a141)